Examples/Nodify.Playground/EditorSettingsView.xaml

```diff
@@ -240,36 +241,48 @@
                       Margin="0 5 5 0" />
 
             <TextBlock Grid.Row="22"
+                       Text="Arrow head end: "
```

Comment:
I would append Connection to this text to make it clear that it works with connections. (maybe there will be a separate section for connection settings in the future)